api/src/main/java/org/eclipse/microprofile/config/Config.java

```diff
@@ -69,13 +69,11 @@
      *
      * @param propertyName
      *             The configuration propertyName.
-     * @return The resolved property value as a String-Optional, which will bypass converters.
+     * @return The resolved property value as a String, which will bypass any converters.
      *
-     * @throws IllegalArgumentException
-     *             is thrown if the propertyName maps to an object that is not a
-     *             String.
+     * 
      */
-    Optional<String> getString(String propertyName);
+    String getRawString(String propertyName);
```

Comment:
Maybe we could avoid the getRawString method in config-1.0 and rethink if we need to standardize this or leave it up to the implementations?